client/package.json

```diff
@@ -28,6 +28,7 @@
 		"react": "^18.1.0",
 		"react-bootstrap": "^2.4.0",
 		"react-dom": "^18.1.0",
+		"react-icons": "^4.4.0",
```

Comment:
We have font-awesome and bootstrap-icon icon installed, it is unnecessary to use another third party icon. Adding may dependencies with the same feature is redundant and can increase bundle size. You have to use font-awesome solid/regular or bootstrap-icon.